platform/spire/src/verify.py

```diff
@@ -353,6 +353,13 @@ def check_user_grant():
 
     print("autogenerated rolebinding for user", repr(authority.UPSTREAM_USER_NAME), "passed basic check!")
 
+@command.wrap
+def check_metallb():
+    "verify that metallb is running"
+    expect_prometheus_query_exact('sum(up{job="metallb-controller"})', 1, "metallb controller is online")
+    expect_prometheus_query_exact('sum(up{job="metallb-speaker"})', 1, "metallb speaker is online")
+    print("metallb is running")
+
```

Comment:
I would really like an automated check that the metallb services aren't just running, but also can handle requests to the exposed IP addresses. This might be too much work to include in this PR, but we should at least file an issue for it.

(This would be similar to our components like dns-monitor and pull-monitor.)

Reply:
Yep, I will file a ticket